common/constant/key.go

```diff
@@ -48,6 +48,9 @@ const (
 	RETRIES_KEY          = "retries"
 	BEAN_NAME            = "bean.name"
 	FAIL_BACK_TASKS_KEY  = "failbacktasks"
+	FORKS_KEY            = "forks"
```

Comment:
Pls add your new config to `config/testdata/` and test it by example.
![image](https://user-images.githubusercontent.com/18375310/62829905-b0ae9c80-bc36-11e9-814c-2208da7c2fa3.png)

Reply:
@fangyincheng make an example? or add fork to testdata and then run success?

Reply:
@fangyincheng done